intake_xarray/netcdf.py

```diff
@@ -1,33 +1,39 @@
 # -*- coding: utf-8 -*-
 import xarray as xr
-from .base import DataSourceMixin
+from .base import XarraySource
 
 
-class NetCDFSource(DataSourceMixin):
-    """Open a xarray file.
+class NetCDFSource(XarraySource):
+    """Open a netcdf file.
+
+    This creates an xarray.Dataset.
+
+    See http://xarray.pydata.org/en/stable/generated/xarray.open_dataset.html
+    for the file formats supported and possible extra arguments.
+
+    NOTE: When reading from OpenDAP URLs do not set the ``chunks`` option to
+    use provided default chunking.
```

Comment:
There is an explanation under the opendap driver that it handles the auth part - should say here explicitly and that the other driver may be necessary